manager/state/raft/raft.go

```diff
@@ -964,8 +967,7 @@ func (n *Node) GetMemberlist() map[uint64]*api.RaftMember {
 		leader := false
 
 		if member.RaftID != n.Config.ID {
-			connState, err := member.Conn.State()
-			if err != nil || connState != grpc.Ready {
+			if err := member.HealthCheck(); err != nil {
```

Comment:
Blocking `GetMemberlist` on waiting for health checks to time out could delay dispatcher startup and make RPCs like `ListNodes` very slow to respond.